sopel/db.py

```diff
@@ -921,7 +936,13 @@ def get_nick_or_channel_value(self, name, key, default=None):
             :meth:`get_channel_value`.
 
         """
-        name = Identifier(name)
+        if not isinstance(name, Identifier):
+            # TODO: log warning/debug?
+            # The goal is to call this method with an existing Identifier
+            # and not a simple string. This is a convenient/shortcut method
+            # and should probably not be used with a simple string anyway.
```

Comment:
Is there a reason it shouldn't be callable with a simple `str`? The standalone methods allow it. For API consistency I would _really_ rather not have this be the only place where passing a `str` instead of `Identifier` throws an error, or even a warning.

Reply:
Yeah you are right, now that SopelDB has an identifier maker, that's doable.